ext/dapr-ext-workflow/dapr/ext/workflow/dapr_workflow_client.py

```diff
@@ -208,33 +208,27 @@ def raise_workflow_event(
         """
         return self.__obj.raise_orchestration_event(instance_id, event_name, data=data)
 
-    def terminate_workflow(
-        self, instance_id: str, *, non_recursive: bool = False, output: Optional[Any] = None
-    ):
+    def terminate_workflow(self, instance_id: str, *, output: Optional[Any] = None):
         """Terminates a running workflow instance and updates its runtime status to
            WorkflowRuntimeStatus.Terminated This method internally enqueues a "terminate" message in
            the task hub. When the task hub worker processes this message, it will update the runtime
            status of the target instance to WorkflowRuntimeStatus.Terminated. You can use
            wait_for_workflow_completion to wait for the instance to reach the terminated state.
 
-           Terminating a workflow by default will terminate all of the child workflows that were started by
-           the workflow instance. If you don't want to terminate child workflows, you can set `non_recursive`
-           flag to true which will disable termination of child workflows.
+           Terminating a workflow by default will terminate all child workflows that were started by
```

Comment:
```suggestion
           Terminating a workflow will terminate all child workflows that were started by
```

It's not configurable anymore, so _by default_ doesn't make sense